### DISCUSSION TITLE
notes/%E5%89%8D%E7%AB%AF/%E6%89%8B%E6%9C%BA%E5%8F%B7%E8%84%B1%E6%95%8F

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#747] [Announcements]
# notes/%E5%89%8D%E7%AB%AF/%E6%89%8B%E6%9C%BA%E5%8F%B7%E8%84%B1%E6%95%8F

创建日期： 2024-01-06 19:45 实例代码 ``` const tel ="13611112222"; //利用正则划分手机号码 const reg = /^(\d{3})\d{3}(\d{4})^/ //利用replae进行替换 const str = tel.replace(reg,'$1**$2'); console.log(str)...

http://localhost:8080/notes/%E5%89%8D%E7%AB%AF/%E6%89%8B%E6%9C%BA%E5%8F%B7%E8%84%B1%E6%95%8F.html

**Comment by zqtop** [comment]
妙啊
